browsers/firefox_android.json

```diff
@@ -54,237 +54,237 @@
         "24": {
           "release_date": "2013-09-17",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/24",
+          "release_notes": "https://www.mozilla.org/firefox/android/24.0beta/releasenotes/",
           "status": "retired"
         },
         "25": {
           "release_date": "2013-10-29",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/25",
+          "release_notes": "https://www.mozilla.org/firefox/android/25.0beta/releasenotes/",
           "status": "retired"
         },
         "26": {
           "release_date": "2013-12-10",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/26",
+          "release_notes": "https://www.mozilla.org/firefox/android/26.0beta/releasenotes/",
           "status": "retired"
         },
         "27": {
           "release_date": "2014-02-04",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/27",
+          "release_notes": "https://www.mozilla.org/firefox/android/27.0beta/releasenotes/",
           "status": "retired"
         },
         "28": {
           "release_date": "2014-03-18",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/28",
+          "release_notes": "https://www.mozilla.org/firefox/android/28.0beta/releasenotes/",
           "status": "retired"
         },
         "29": {
           "release_date": "2014-04-29",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/29",
+          "release_notes": "https://www.mozilla.org/firefox/android/29.0beta/releasenotes/",
           "status": "retired"
         },
         "30": {
           "release_date": "2014-06-10",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/30",
+          "release_notes": "https://www.mozilla.org/firefox/android/30.0beta/releasenotes/",
           "status": "retired"
         },
         "31": {
           "release_date": "2014-07-22",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/31",
+          "release_notes": "https://www.mozilla.org/firefox/android/31.0beta/releasenotes/",
           "status": "retired"
         },
         "32": {
           "release_date": "2014-09-02",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/32",
+          "release_notes": "https://www.mozilla.org/firefox/android/32.0beta/releasenotes/",
           "status": "retired"
         },
         "33": {
           "release_date": "2014-10-14",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/33",
+          "release_notes": "https://www.mozilla.org/firefox/android/33.0beta/releasenotes/",
           "status": "retired"
         },
         "34": {
           "release_date": "2014-12-01",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/34",
+          "release_notes": "https://www.mozilla.org/firefox/android/34.0beta/releasenotes/",
```

Comment:
This shoudn’t point to the beta release notes, but to the [normal release notes](https://www.mozilla.org/firefox/android/34.0/releasenotes/).